### PR TITLE
Stabilise UI tests changing positions

### LINKED ISSUE #35535: DragNDrop and Tests UI on the migrated page
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

In UI Tests (`tests/UI/campaigns/functional/BO/03_catalog/04_attributesAndFeatures/01_attributes/02_changePosition.ts`), 
* when I drag the first element on the third, the first becomes the third :heavy_check_mark: 
* when I drag the third element on the first, the third become the second :red_circle: 

It's the same thing on Attribute Value page. (and all other of migrated page with dragndrop rows except for Categories page which works but doesn't use the same component in the grid).

### Expected behavior

Adapt all the grid 
* when I drag the first element on the third, the first becomes the third :heavy_check_mark: 
* when I drag the first element on the second, the first becomes the first :heavy_check_mark: 

### Steps to reproduce

1. Go to BO
2. Go to "Catalog > Attributes & Features" page (migrated)
3.  =>
* when I drag the first element on the third, the first becomes the third :heavy_check_mark: 
* when I drag the third element on the first, the third become the second :red_circle: 

### PrestaShop version(s) where the bug happened

9.0.x

### PHP version(s) where the bug happened

8.2

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

@PrestaShopCorp

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Change the drag and drop method used to modify the positions to make it more stable
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green and UI tests green
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/11481569116
| Fixed issue or discussion?     | Fixes #35535
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]
:heart: @jolelievre :heart:

**Review by nesrineabdmouleh — DISMISSED** [review]

**Review by PululuK — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by PululuK — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
